npc/common/src/main/java/org/screamingsandals/lib/npc/NPCImpl.java

```diff
@@ -60,8 +65,27 @@
     private volatile boolean lookAtPlayer;
     private final List<MetadataItem> metadata;
 
+    
     private SClientboundSetPlayerTeamPacket.CollisionRule collisionRule;
-
+    private static boolean IS_BUNGEECORD_ENABLED;
+    static {
+        File spigotYmlFile = new File("spigot.yml");
+        IS_BUNGEECORD_ENABLED = false;
+        java.util.regex.Pattern pattern = Pattern.compile("bungeecord: ?(y|yes|true|on)",Pattern.CASE_INSENSITIVE);
```

Comment:
remove redundant `java.util.regex`, you have `Pattern` already imported

make the group non-capturing: `bungeecord: ?(?:y|yes|true|on)`

add a space after the comma